<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_WorkWid</class>
 <widget class="QWidget" name="Home_WorkWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>状态显示区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_33">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>产品类型</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="devLab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>245</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_34">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>设备序号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="snLab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_36">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>MAC地址</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="macEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>245</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_37">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>已被使用：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="macCntLab">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_35">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>客户名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="userEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>245</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_38">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>剩余数量：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="cntSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QTextEdit" name="textEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>260</height>
      </size>
     </property>
     <property name="title">
      <string>计时区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="timeLab">
        <property name="text">
         <string>未开始</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>计数区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_27">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>数量:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="allLcd">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_28">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>失败:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLCDNumber" name="errLcd">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>35</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_29">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>通过:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="okLcd">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_30">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>合格率:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLCDNumber" name="passLcd">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>设置区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="setBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="outputBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>输出位阈值</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="snCheckBox">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>创建设备序列号</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>操作区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>设备</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="startBtn">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>开始设置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="typeComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>SI/BM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-PDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZPDU特殊定制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>标准RPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-BUSBAR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPDU一期/特殊定制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-PDU特殊定制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZPDU华为</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZPDU标准</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
